bedrock/tabzilla/templates/tabzilla/tabzilla.js

```diff
@@ -1,729 +0,0 @@
-        }
-
-        return false;
-    };
-    // Expose the object for the tests
-    Tabzilla.infobar = Infobar;
-    var addEaseInOut = function () {
-        $.extend($.easing, {
-            'easeInOut':  function (x, t, b, c, d) {
-                if (( t /= d / 2) < 1) {
-                    return c / 2 * t * t + b;
-                }
-                return -c / 2 * ((--t) * (t - 2) - 1) + b;
-            }
-        });
-    };
-    var addMatchMediaPolyfill = function () {
-        window.matchMedia = window.matchMedia || (function( doc, undefined ) {
-            var bool;
-            var docElem = doc.documentElement;
-            var refNode = docElem.firstElementChild || docElem.firstChild;
-            // fakeBody required for <FF4 when executed in <head>
-            var fakeBody = doc.createElement( "body" );
-            var div = doc.createElement( "div" );
-
-            div.id = "mq-test-1";
-            div.style.cssText = "position:absolute;top:-100em";
-            fakeBody.style.background = "none";
-            fakeBody.appendChild(div);
-
-            return function(q){
-                div.innerHTML = "&shy;<style media=\"" + q + "\"> #mq-test-1 { width: 42px; }</style>";
-                docElem.insertBefore( fakeBody, refNode );
-                bool = div.offsetWidth === 42;
-                docElem.removeChild( fakeBody );
-                return {
-                    matches: bool,
-                    media: q
-                };
-            };
-        }( document ));
-    };
-    var init = function () {
-        $('body').prepend(content);
-        tab = $('#tabzilla');
-        panel = $('#tabzilla-panel');
-        nav = $('#tabzilla-nav');
-        headlines = nav.find('h2');
-
-        if (isIE9 && !hasMediaQueries) {
-            addMatchMediaPolyfill();
-            hasMediaQueries = true;
-        }
-
-        addEaseInOut();
-
-        checkMode();
-        $(window).on('resize', function () {
-            checkMode();
-        });
-
-        panel.on('keydown', function (event) {
-            if (event.which === 27) {
-                event.preventDefault();
-                Tabzilla.close();
-                tab.focus();
-            }
-        });
-
-        tab.attr('aria-label', '{{ _('Mozilla links')|js_escape }}');
-
-        tab.on('click', function (event) {
-            event.preventDefault();
-            if (opened) {
-                Tabzilla.close();
-            } else {
-                Tabzilla.open();
-            }
-        });
-
-        $.each((tab.data('infobar') || '').split(' '), function (index, value) {
-            var setup = Infobar[value];
-            if (setup) {
-                setup.call();
-            }
-        });
-
-        // Careers teaser in error console.
-        $(window).load(function() {
-            try {
-                console.log("{% filter js_escape|safe %}{% include "includes/tabzilla-console-footer.html" %}{% endfilter %}");
```

Comment:
Can we remove `includes/tabzilla-console-footer.html` in this PR too?

Reply:
I think that might be the same one that's included in the head of every page? Might not be though. I'll check.

Reply:
Yup. Deleted. Good catch.